### DISCUSSION TITLE
Any chance for a 4.1.1 release?

### DISCUSSION THREAD

**Discussion by mbklein** [openseadragon/openseadragon#2501] [General]
I'm the maintainer of [samvera/serverless-iiif](https://github.com/samvera/serverless-iiif), and I've been working through some bug reports related to OSD 4.1.0 compatibility. After running a few local tests, I've found that the issue was fixed by #2416 back in October. But without an OSD release that includes the fix, I'm at a loss for what to tell my users,. It doesn't seem to be something I can fix on the server side without compromising other features. Is there any chance you might be willing to push a v4.1.1 release so that there's an easy fix for these users?

**Comment by iangilman** [comment]
Interesting. I'll have to think about how we might accomplish that. I suppose it's possible.

**Reply by mnyrop** [reply]
Hi @iangilman, jumping in to agree with @mbklein and add supporting tests: https://mnyrop.github.io/osd-region-bug-demo/osd/v4-1-0/

[HEAD](https://github.com/openseadragon/openseadragon/tree/master) (which includes the fix from #2416) is working for me as well: https://mnyrop.github.io/osd-region-bug-demo/osd/head/

So far I've just steered clear of OSD v4.1.0, but now Mirador's demo site (and their [v4.0-alpha releases](https://mnyrop.github.io/osd-region-bug-demo/mirador/v4-alpha/) are using OSD v4.1.0 making the issue more difficult to avoid. I can submit a ticket to Mirador requesting they roll back to OSD v3.1 or 4.0, but it would be a huge help to instead bump them up to an OSD v4.1.1 with the fix included.

> *Update: Mirador ticket added ProjectMirador/mirador/issues/3897*

**Reply by mbklein** [reply]
> Mirador's demo site (and their [v4.0-alpha releases](https://mnyrop.github.io/osd-region-bug-demo/mirador/v4-alpha/) are using OSD v4.1.0

More to the point, their `package.json` file specifies `"openseadragon": "^2.4.2 || ^3.0.0 || ^4.0.0"`, meaning a corrective v4 release would fix the issue for Mirador and anything downstream of Mirador with a simple `npm update`.

**Reply by iangilman** [reply]
Excellent points! Okay, I believe I have done it: 

https://github.com/openseadragon/openseadragon/releases/tag/v4.1.1

Let me know if that works!

**Reply by mbklein** [reply]
Fantastic! Thank you so much.
